cmd/ctr/commands/images/pull.go

```diff
@@ -78,6 +82,10 @@ command. As part of this process, we do the following:
 		if err != nil {
 			return err
 		}
+		if context.Bool("all-manifests") {
```

Comment:
This shouldn't be needed, as line 81 (`content.NewFetchConfig(ctx, context)`) should manage it.

Reply:
Fixed.